schema-next.yaml

```diff
@@ -4,6 +4,10 @@ versions:
   next:
     all:
       changes:
+        # https://github.com/open-telemetry/semantic-conventions/pull/1216
+        - rename_attributes:
+          attribute_map:
+            tls.client.server_name: server.address
```

Comment:
The rename attribute here is because _when_ used, tls instrumentations should report `server.address` instead, right? Because since this attribute only existed in the registry we don't add now the `server.address` in `tls.yaml` so it's more of a delete than a rename?.

Reply:
Yes, that's what I meant, but one could also see as delete, since it was never part of any convention that used it (it was only on the registry), but I guess this makes sense.